nova/conf/vmware.py

```diff
@@ -158,6 +158,12 @@
 For spawning a VM on a prepared empty host, we need all other VMs to be in a
 DRS VM group that prohibits them from running on the prepared host. This
 setting configures the name of that VM group.
+"""),
+    cfg.BoolOpt('forward_instance_logs_to_loginsight',
```

Comment:
I'd name it `mirror_instance_logs_to_syslog`, because loginsight is just  a backend receiving the syslog of the host